sunpy/image/transform.py

```diff
@@ -41,10 +46,19 @@ def affine_transform(image, rmatrix, order=3, scale=1.0, image_center=None,
         Defaults to `True` i.e., recenter to the center of the array.
     missing : `float`, optional
         The value to replace any missing data after the transformation.
-    use_scipy : `bool`, optional
-        Force use of `scipy.ndimage.affine_transform`.
-        Will set all "NaNs" in image to zero before doing the transform.
-        Defaults to `False`, unless scikit-image can't be imported.
+    method : `string` or function(), optional
+        1. If `string`: `skimage`, `scipy`, or `cv2'
+        If `skimage`, uses :func:`skimage.transform.warp`.
+        If `scipy`, uses :func:`scipy.ndimage.interpolation.affine_transform`.
+        If `cv2`, uses :func:`cv2.warpAffine`.
+        2. Elif function, uses user-defined function to perform affine transform.
+        See `notes` for function requirements.
+        Default: `none`: Will attempt to use :func:`skimage.transform.warp`;
+        on ImportError, will use :func:`scipy.ndimage.interpolation.affine_transform`.
+        (This behavior is identical to the now-deprecated `use_scipy=False`)
+        (Option `none` will be deprecated in future releases. Please explicitly set method.)
```

Comment:
Can the default be `'scipy'` instead of `'none'`, to avoid us having to deprecate this in the future?

Reply:
Another issue (which I've just realized) is that the current default behavior of `rotate` is to attempt `skimage`, but on an `ImportError`, defaults to `scipy`. I have kept this functionality, but this will have to be deprecated in the future in favor of explicitly setting method each time.